pkg/image/apis/image/helper.go

```diff
@@ -41,6 +46,16 @@ const (
 	ImportRegistryNotAllowed = "registry is not allowed for import"
 )
 
+// knownSignatureStores is a list of known signature stores. The key represents
+// a registry name and it maps to a signature store for that registry. The
+// signature store has to be HTTPS and it must offer image signatures based on
+// this format: https://STORE_URL/IMAGE_NAME/signature-1
+// The response must be valid image signature blob.
+// TODO: Maybe this should be configurable via master config?
```

Comment:
> Is it possible for non-admin tenants to create their own image streams, choosing their own external registries?

Yes, it's possible, but right now only cluster-priv signers can add signatures to the API. So this is far from tenant enabled. I believe we would address this comprehensively at a later date.

@mfojtik can you confirm the image importer will need **system:image-signer** role?

> ...and without recompiling OpenShift.

*restarting* OpenShift.